src/mscorlib/corefx/System/Globalization/CultureData.Unix.cs

```diff
@@ -180,8 +198,10 @@ private static CultureInfo GetUserDefaultCulture()
 
         private static bool IsCustomCultureId(int cultureId)
         {
-            // TODO: Implement this fully.
-            return false;
+            const int LOCALE_CUSTOM_DEFAULT = 0x0c00;
+            const int LOCALE_CUSTOM_UNSPECIFIED = 0x1000;
+
+            return (cultureId == LOCALE_CUSTOM_DEFAULT || cultureId == LOCALE_CUSTOM_UNSPECIFIED);
```

Comment:
I assume that this means that ICU is never going to return one of things numbers an LCID for an existing culture, i.e. it doesn't use this when there's no corresponding windows LCID for a culture?